### ISSUE TITLE
jsonassets crop pack idea

### ISSUE THREAD

**Issue by Traehgniw** [StardewModders/mod-ideas#280]
stuff like lamb's quarter and farmed dandelion and other things that used to be common crops or food plants and are now often considered weeds

**Comment by KarBoy2314PL** [comment]
please split your ideas into multiple issues

**Comment by Traehgniw** [comment]
But this was one idea? "JsonAssets crop pack of thingst that used to be common food crops and are now often considered weeds", with two examples.

**Comment by notwondering** [comment]
I'd like to take this on if I can! Feel free to snatch it from me if I don't post something in 3 weeks. The base game adds regular dandelion now (not sure if it did when you first posted this), but I'm looking at Purslane, Lamb's Quarters, Good King Henry, Agrimony, and a few others. Please post here if anyone thinks I'm going to have a conflict with another mod (I'm checking Mouse's PPJA list, Seasonal Wildflowers on Nexus/anything else on Nexus that looks similar, and the giant PPJA spreadsheet for conflicts already).

Also, I have trouble with art. If anyone wants to volunteer to sprite these, post here please. I can definitely do the JA writing, just having trouble learning how to mess around in spriting programs atm.

**Comment by minervamaga** [comment]
@notwondering they meant a dandelion crop I believe.  I'll assign you

**Comment by Traehgniw** [comment]
since Multi Yield Crops exists you could differentiate farmed dandelion from wild dandelion by having the dandelion crop also give dandelion roots, possibly?

**Comment by notwondering** [comment]
True! I personally have had an issue with MYC where I can harvest an infinite number of the secondary crop and would like to either understand that or have a fix for it before I attempt to write anything with it, but definitely something to keep in mind :-) Thanks!

**Comment by Traehgniw** [comment]
i think that needs a bug report on MYC's page!

**Comment by notwondering** [comment]
Will do. I saw someone had already put it in the comments for the only crop currently using MYC (mulberry), so I was lazy about escalating.

**Comment by notwondering** [comment]
So far, Agrimony works as written and sprite work is as difficult as expected. If you have any other specific requests (aside from lamb's quarter and dandelion) for inclusion in this mod, lemme know. I think I will pause at around 10 crops for first release and potentially update if I feel like continuing to sprite more crops.

**Comment by tiakall** [comment]
Crops are now done in CP - I don't know of any packs with that theme or with either of those two specific crops, though.
